src/react_native/pure.cljs

```diff
@@ -0,0 +1,12 @@
+(ns react-native.pure
+  (:require ["react-native" :refer (AppRegistry View Text SafeAreaView TouchableHighlight FlatList Fragment Pressable)]
+            ["react" :refer (useState useEffect) :as React]))
+
+(def view       (.createFactory React View))
```

Comment:
The `createFactory` method would be removed in the future

https://react.dev/reference/react/createFactory

Reply:
We can replace it with `createElement`:

https://react.dev/reference/react/createFactory#copying-createfactory-into-your-project

Reply:
```cljs
(defn create-factory [type]
  (.bind React/createElement js/React type))
 ```

Reply:
yes, thanks. its just a quick implementation for experiments